docs/process/BranchStrategy.md

```diff
@@ -0,0 +1,35 @@
+# Overview
+
+**Branch Strategy for Issue** defines how should you create/maintain/merge a feature branch when a new task is assigned to you.  
+A feature branch should be named as *your account*/*issue id*/*description*. Or in case you use Azure DevOps for manage tasks, task/*task id*/*description*.
+
+## Branch Strategy for Tasks
```

Comment:
Is this supposed to be branching strategy for this repo or in general? 
Maybe we should call out we are using "github flow" (vs. "git flow")?

Reply:
added.